framework/src/test/java/org/tron/core/jsonrpc/ApiUtilTest.java

```diff
@@ -16,6 +19,17 @@
 
 public class ApiUtilTest {
 
+
+  @BeforeClass
```

Comment:
Is there any benefit that add init and clear methods ? It seems no component used in this case.

Reply:
Using the correct configuration instead of reusing the configuration from the previous unit test resulted in an error.